docs/getting-started/ingest-data.asciidoc

```diff
@@ -3,27 +3,30 @@
 
 To ingest data, you can use:
 
-* The Elastic Agent with the **Elastic Endpoint Integration**, which protects
-your hosts and sends logs, metrics, and endpoint security data to {es-sec}
-(see <<install-endpoint>>).
+* The {fleet-guide}/fleet-overview.html[{agent}] with the **Elastic Endpoint Integration**, which protects
+your hosts and sends logs, metrics, and endpoint security data to {elastic-sec}. See, <<install-endpoint>>.
+* The {agent} with integrations, which are made available through the {fleet-guide}/fleet-overview.html#package-registry-intro[Elastic Package Registry (EPR)]. To install an integration that works with {elastic-sec}, go to the Home page and click *Add integrations*. On the Integrations page, click the *Security* category filter, then select an integration and follow the guided steps to install it.
+
++
+TIP: For more information on integrations, refer to {integrations-docs}[{integrations}].
```

Comment:
I don't think this needs to be a Tip admonition; it's just a link to another docs page. This could be "See [link]." added to the end of the previous paragraph, just like some of the other bullets in this list.

Reply:
Agree!